x/gov/migrations/v5/store.go

```diff
@@ -36,7 +36,7 @@ func MigrateStore(ctx sdk.Context, storeService corestoretypes.KVStoreService, c
 	}
 
 	defaultParams := govv1.DefaultParams()
-	params.ExpeditedMinDeposit = defaultParams.ExpeditedMinDeposit
+	params.ExpeditedMinDeposit = params.MinDeposit // Use regular `min_deposit` as `expedited_min_deposit`
```

Comment:
We update these other params by though updated value of `defaultParams := govv1.DefaultParams()`. I added a comment to highlight

Only `ExpeditedMinDeposit` needs to be explicitly updated here since we need the `min_deposit` value from state.

Reply:
makes sense. can we update the comment to highlight "need to use `min_deposit` from state to use the same denom"?

the current comment doesn't explain why we are doing this